src/js/structs/ServiceTree.js

```diff
@@ -91,6 +102,16 @@ module.exports = class ServiceTree extends Tree {
     });
   }
 
+  getInstances() {
```

Comment:
can we name this `getInstanceCount` or `getInstancesCount`? (I don't know how to english) since this is a number